### PR TITLE
Update markdown.md

### PR DESCRIPTION
**Pull request by Gogiu23**
i change how to iclude img in markdown because in the way was made doesn't work

### CODE REVIEW

**Comment by syntax-tm** [comment]
## Purpose

This example is a single reference-style image. The `[img]` at the end of the line is the [link label](https://github.github.com/gfm/#link-label). That link label refers to a [link reference definition](https://github.github.com/gfm/#link-reference-definition) that can be found on the second line.

```md
![Image alt text][img]
[img]: http://foo.com/img.jpg
```

## Proposed Changes

With your changes the first line is no longer valid because the link label `img` no longer has a matching reference definition.

```md
![Image alt text][img]
```

The second line is just an inline image whose `alt` text will be set to `img`.

```md
![img](https://foo.com/img.jpg)
```

So, this example of a reference-style link is now an invalid link since no matching definition exists and the second line is just a duplicate example of inline syntax.

## The Fix

A [reference link](https://github.github.com/gfm/#reference-link) is comprised of the [link text](https://github.github.com/gfm/#link-text) and the [link label](https://github.github.com/gfm/#link-label). That link label references a [link reference definition](https://github.github.com/gfm/#link-reference-definition) located "elsewhere in the document."

```md
![Image alt text][img]

[img]: http://foo.com/img.jpg
```

Basically, you just need to add at least one empty line between the image and the link definition.

## Example

![GitHub Logo][github_logo_2]

[github_logo_2]: https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png

```md
![GitHub Logo][github_logo_2]

[github_logo_2]: https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png
```
